lambdas.py

```diff
@@ -0,0 +1,33 @@
+import json
+import logging
+
+from apig_wsgi import make_lambda_handler
+
+from webapp import Config, create_app
+
+logger = logging.getLogger(__name__)
+CONFIG = Config()
+
+
+def lambda_handler(event: dict, context: dict) -> dict:
+    """Launches the Flask app when the Lambda function is invoked.
+
+    This is achieved by using 'apig-wsgi' to wrap the Flask app in an
+    AWS Lambda handler function. A WSGI is responsible for carrying out
+    the communication between the web server (Lambda) and the Python
+    application (the Flask app).
+
+    See https://github.com/adamchainz/apig-wsgi/tree/main.
+    """
```

Comment:
😍 

Overall, looks really good, thanks for adding.  This is a tricky thing to succinctly describe.

**Completely optional**: because this repository / docstring may be a template for future flask apps, I might propose a couple of small tweaks.

Maybe something along the lines of:
```python
"""Launches the Flask app when the Lambda function is invoked.
    
This is achieved by using the library 'apig-wsgi' to wrap the Flask app in an
AWS Lambda handler function. This wrapper converts the Function URL 
request payload into a WSGI object for the Flask app, then converts the 
Flask app response into a payload suitable for the Lambda Function URL
to return to the caller.

See https://github.com/adamchainz/apig-wsgi/tree/main.
"""
```

But I'm unsure if this actually helps or hurts.  I'm happy with the description as-is if you'd like to proceed with it!

Reply:
I like the additional detail! I updated the docstring.